README.md

```diff
@@ -172,6 +172,19 @@ deathRecord.DeathDay = -1;
 deathRecord.DeathTime = "-1";
 ```
 
+#### Names in FHIR
+
+FHIR manages names in a way that there is a fundamental incompatibility with IJE: in FHIR the
+"middle name" is stored as the second element in an array of given names. That means that it's not
+possible to set a middle name without first setting a first name. The library handles this by
+
+1. Requiring the entire given name (first name and any middle names) to be set all at once when
```

Comment:
I expected there to be some changes in one of the DeathRecord files to enforce this, am I missing it?